core.server.ts

```diff
@@ -177,10 +177,22 @@ function createProductionRequestHandler(): RequestHandler {
 
 // Create a request handler that watches for changes to the server build during development.
 function createDevRequestHandler(): RequestHandler {
+   // setup server RDT (Remix Development Tools)
+   const {
```

Comment:
Without the if check this will bundle it into production, if you use this server there too! Careful!

Reply:
I think it should be fine since createDevRequestHandler is only called here

```jsx

   app.all(
      "*",
      process.env.NODE_ENV === "development"
         ? createDevRequestHandler()
         : createProductionRequestHandler(),
   );
```